src/eeg.jl

```diff
@@ -16,7 +17,7 @@ end
 Attributes:
 
 * `positions::Vector{<: Point} = Makie.automatic`: Can be calculated from label (channel) names. Currently, only 10/20 montage has default coordinates provided.
-
+* `labels::Vector{<: String} = Makie.automatic`: Add custom labels, in case `label_text` is set to true. If `positions` is not specified, `labels` are used to look up the 10/20 coordinates.

```

Comment:
```suggestion
* `labels::AbstractVector{<:AbstractString} = Makie.automatic`: Add custom labels, in case `label_text` is set to true. If `positions` is not specified, `labels` are used to look up the 10/20 coordinates.
```